### DISCUSSION TITLE
Virtual Tour - Can I dynamically prevent navigation to a new panorama?

### DISCUSSION THREAD

**Discussion by elmonty** [mistic100/Photo-Sphere-Viewer#1223] [General]
If I want to prevent navigation to a new area until some other task has been completed, is that possible? For example, could I respond to an event on the 3D arrows and pop up an alert that tells the user they have to look at something else first before they may continue?

**Comment by mistic100** [comment]
That's not possible. You will have to drop Virtual Tour plugin and implement your own navigation markers and logic.

**Reply by elmonty** [reply]
That's too bad. How can I create markers like the 3D arrows that rotate with the user's perspective?

**Comment by elmonty** [comment]
Alternatively, would it possible to dynamically enable/disable or hide/show arrows? Or have some kind of cancelable select event?
